.github/workflows/build_hub.yml

```diff
@@ -27,15 +27,15 @@ jobs:
               - 'prime-router/**'
               - '.github/workflows/build_hub.yml'
 
-  build_router:
+  unit_tests:
     name: Build Router
     needs: pre_job
     if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
     runs-on: ubuntu-latest
     env:
       # These are for CI and not credentials of any system
-      POSTGRES_USER: prime
-      POSTGRES_PASSWORD: changeIT!
+      DB_USER: prime
+      DB_PASSWORD: changeIT!
```

Comment:
Why was this changed?

Reply:
It allows the removal of the GITHUB_ENV call later.

Reply:
eventually we could use github environments to nicely store some of these settings and reduce the conditionals.